src/olympia/devhub/api_views.py

```diff
@@ -0,0 +1,28 @@
+from olympia import amo
+from olympia.addons.views import AddonVersionViewSet
+from olympia.api.permissions import (
+    AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted, AnyOf)
+from olympia.devhub.models import ActivityLog
+from olympia.devhub.serializers import ActivityLogSerializer
+from olympia.versions.models import Version
+
+
+class VersionReviewNotesViewSet(AddonVersionViewSet):
+    permission_classes = [
+        AnyOf(AllowAddonAuthor, AllowReviewer, AllowReviewerUnlisted),
+    ]
+    serializer_class = ActivityLogSerializer
+    queryset = ActivityLog.objects.all()
+    filter = amo.LOG_REVIEW_QUEUE_DEVELOPER
+
+    def get_queryset(self):
+        self.get_addon_object()  # Just to force the permission checks.
+        if not hasattr(self, 'version_object'):
+            self.version_object = Version.unfiltered.all().get(
+                pk=self.kwargs['version_pk'])
+
+        alog = ActivityLog.objects.for_version(self.version_object)
+        if self.filter:
```

Comment:
No, I was going to make it more general ActivityLog viewset where you could specify the filter but changed my mind.